scripts/diff_to_added_lines.py

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
```

Comment:
Also, a general question: Should we start porting our internal scripts to Python 3 instead? I don't think we should be maintaining Python 2 scripts forever.

Reply:
Yes, I do think that all scripts should be ported to Python 3.